buildSrc/src/main/kotlin/Config.kt

```diff
@@ -5,9 +5,9 @@ object Config {
     private const val kotlinVersion = "1.7.10"
 
     object SdkVersions {
-        const val compile = 29
-        const val target = 29
-        const val min = 16
+        const val compile = 34
+        const val target = 34
+        const val min = 21
```

Comment:
The latest version of Auth requires minSdk 23, so we might have to bump this one a little higher 😅